WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductVariationFormViewModel.swift

```diff
@@ -272,6 +276,7 @@ extension ProductVariationFormViewModel {
         let deleteAction = ProductVariationAction.deleteProductVariation(productVariation: productVariation.productVariation) { result in
             switch result {
             case .success:
+                self.onVariationDeletion?(self.productVariation.productVariation)
```

Comment:
nit: maybe weak self just in case?

Reply:
nit: could this be unit tested? like mocking `ProductVariationAction.deleteProductVariation` to return success and asserting the callback is called, while the error case doesn't